packages/react-components/react-alert/Spec.md

```diff
@@ -81,9 +99,11 @@ _Explain how the component will behave in use, including:_
   - _Touch_
   - _Screen readers_
 
+TODO - What goes here?
```

Comment:
Here go interaction considerations for the component, for example:
- Does it change styling when hovered?
- Where does focus go when tabbing to it?

Among similar things for touch and state changes.